fdbclient/PaxosConfigTransaction.actor.cpp

```diff
@@ -151,9 +172,10 @@ class GetGenerationQuorum {
 			} catch (Error& e) {
 				if (e.code() == error_code_failed_to_reach_quorum) {
 					TEST(true); // Failed to reach quorum getting generation
-					wait(delayJittered(0.01 * (1 << retries)));
+					wait(delayJittered(0.005 * (1 << retries)));
```

Comment:
I think this can cause the delay to grow so large after just a couple retries that the `delayJittered` doesn't have much of an effect on two nodes contending for a quorum.

Reply:
See https://github.com/apple/foundationdb/pull/6353#discussion_r802179184.